### DISCUSSION TITLE
Plantuml embedded hyperlink not interactive

### DISCUSSION THREAD

**Discussion by mlafrance-complianceds** [requarks/wiki#3597] [Error / Bug Report]
**Describe the bug**
When using the plantuml syntax to embed a hyperlink in an activity diagram, the link itself when rendered into the graphic is not interactive ie I cannot click it.

**To Reproduce**
1. use markdown editor
2. create a plantuml activity diagram
(sample code to use)

```plantuml
start
[[http://plantuml.com]]:Some activity;
:Some [[http://plantuml.com link]];
end
```

3. In viewer try to click link. It is not interactive.

**Expected behavior**
When clicking the link it should take action to direct to the specified URL

**Host Info (please complete the following information):**
 - OS: Linux - CentOS (core) 8.1.1911 x64
 - Wiki.js version: 2..4.105
 - Database engine: Microsoft SQL Server 2017

**Additional context**
https://plantuml.com/link

**Comment by bharatrajagopalan** [comment]
@mlafrance-complianceds  I think this is what you are looking for?

https://github.com/Requarks/wiki/pull/1337

**Comment by zhuzhzh** [comment]
As the early pr is not merged,  is there any workaround based on the current release?

**Comment by oneacik** [comment]
Pinging this issue.
Anything to be done for [this PR](https://github.com/requarks/wiki/pull/2754) to be merged?
It is quite important for me so I could invest a moment to finish it.

**Reply by oneacik** [reply]
@NGPixel
